homeworkW13D5/src/homeworkW13D5/ORM/Model.java

```diff
@@ -0,0 +1,21 @@
+package homeworkW13D5.ORM;
+public abstract class Model {
+
+	@SuppressWarnings("unused")
+	private String tableName;
+
+	public Model(String tableName) {
+		this.tableName = tableName;
+	}
+
+	public String findByPk(Integer id, String tableName) {
+		return String
+				.format("SELECT * FROM %s  WHERE id = %d ;", tableName, id);
+	}
+
+	public String findByAttribute(String tableName, String value) {
```

Comment:
Same issue again. Method should take attribute name/value argument. Table name is already known to this class.